frontend/src/components/config/Theme.tsx

```diff
@@ -29,6 +30,7 @@ export const ThemeConfig: any = {
     xSmall: '0.4rem',
     small: '0.6rem',
     mediumSmall: '0.8rem',
+    xMediumSmall: '0.9rem',
```

Comment:
Perhaps just call this "medium"?

Reply:
Sure sounds good - as a side note this will put medium as a smaller size than default, followed by mediumLarge

Reply:
Yeah I'm ok with that if you are, sounds good 👍